front/temporal/scrub_workspace/workflows_queue_v1.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  ParentClosePolicy,
+  proxyActivities,
+  setHandler,
+  sleep,
+  startChild,
+} from "@temporalio/workflow";
+
+import type * as activities from "./activities";
+import { runScrubFreeEndedWorkspacesSignal } from "./signals";
+
+const { shouldStillScrubData } = proxyActivities<typeof activities>({
+  startToCloseTimeout: "1 minutes",
+});
+const { sendDataDeletionEmail } = proxyActivities<typeof activities>({
+  startToCloseTimeout: "1 minutes",
+  retry: {
+    // We really want to avoid sending the email infinitely.
+    maximumAttempts: 1,
+  },
+});
+
+const { endSubscriptionFreeEndedWorkspacesActivity } = proxyActivities<
+  typeof activities
+>({
+  startToCloseTimeout: "5 minutes",
+});
+
+const { scrubWorkspaceData, pauseAllConnectors, pauseAllTriggers } =
+  proxyActivities<typeof activities>({
+    startToCloseTimeout: "60 minutes",
+  });
+
+export async function scheduleWorkspaceScrubWorkflowV2({
+  workspaceId,
+}: {
+  workspaceId: string;
+}): Promise<boolean> {
+  await pauseAllConnectors({ workspaceId });
```

Comment:
... here, we remove the `if` from [here](https://github.com/dust-tt/dust/blob/mainfront/temporal/scrub_workspace/workflows.ts#L39-L38)